internal/converter/types.go

```diff
@@ -548,8 +554,12 @@ func (c *Converter) recursiveConvertMessageType(curPkg *ProtoPackage, msgDesc *d
 			}
 		case "Duration":
 			jsonSchemaType.Type = gojsonschema.TYPE_STRING
+		case "Empty":
+			jsonSchemaType.Type = gojsonschema.TYPE_OBJECT
+			jsonSchemaType.Format = "empty"
 		case "Struct":
 			jsonSchemaType.Type = gojsonschema.TYPE_OBJECT
+			jsonSchemaType.Format = "struct"
```

Comment:
Nice, so we're just adding additional information wihout changing the existing type: "object" so this is backwards compatible.